### PR TITLE
Branch rename: Change all references to the master branch to point to main instead.

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: service/iot-service-samples/digitaltwin-service-samples/thermostat-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/Thermostat.java [hunk 1]
```diff
@@ -170,7 +170,7 @@ private static void InvokeMethodOnRootLevel() throws IOException, InterruptedExc
                                 System.out.println("Invoked Command " + commandName + " failed: " + error);
                             }
                             else {
-                                System.out.println("Ensure the device sample is running for this sample to succeed - https://github.com/Azure/azure-iot-sdk-java/tree/master/device/iot-device-samples/pnp-device-sample/thermostat-device-sample.");
+                                System.out.println("Ensure the device sample is running for this sample to succeed - https://github.com/Azure/azure-iot-sdk-java/tree/mamainster/device/iot-device-samples/pnp-device-sample/thermostat-device-sample.");
```
Comment (timtay-microsoft) [inline service/iot-service-samples/digitaltwin-service-samples/thermostat-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/Thermostat.java:173]: This got a bit messed up

Comment (azabbasi) [reply service/iot-service-samples/digitaltwin-service-samples/thermostat-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/Thermostat.java:173]: oops

Comment (timtay-microsoft) [reply service/iot-service-samples/digitaltwin-service-samples/thermostat-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/Thermostat.java:173]: "Mainster" would be a cool default branch name, though

**Review by timtay-microsoft — APPROVED** [review]
Just the one slip up, otherwise looks good

**Review by drwill-ms — APPROVED** [review]

**Pull request merged**